introduction.md

```diff
@@ -0,0 +1 @@
+Introduction test file
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
http://campus.codeschool.com/courses/mastering-github/level/1/section/3/pull-request